Firestore/core/src/remote/serializer.cc

```diff
@@ -1429,9 +1430,29 @@ std::unique_ptr<WatchChange> Serializer::DecodeDocumentRemove(
 
 std::unique_ptr<WatchChange> Serializer::DecodeExistenceFilterWatchChange(
     ReadContext*, const google_firestore_v1_ExistenceFilter& filter) const {
-  ExistenceFilter existence_filter{filter.count};
-  return absl::make_unique<ExistenceFilterWatchChange>(existence_filter,
-                                                       filter.target_id);
+  ExistenceFilter existence_filter = DecodeExistenceFilter(filter);
+  return absl::make_unique<ExistenceFilterWatchChange>(
+      std::move(existence_filter), filter.target_id);
+}
+
+ExistenceFilter Serializer::DecodeExistenceFilter(
+    const google_firestore_v1_ExistenceFilter& filter) const {
+  // Create bloom filter if there is an unchanged_names present in the filter
+  // and inputs are valid, otherwise keep it null.
+  absl::optional<BloomFilter> bloom_filter = absl::nullopt;
```

Comment:
Is it possible to omit `= absl::nullopt` in this line? I think it ma be redundant, because I think that a default-constructed `absl::optional` object is "empty".